.prettierignore

```diff
@@ -9,3 +9,6 @@
 **/wpt
 **/manifest.json
 **/wptreport.json
+
+# Lock files
+**/*.lock
```

Comment:
I'm assuming this is to stop `flake.lock` from being formatted? Or were there other `.lock` files that prettier was formatting?

Reply:
Yes, in this case it was the `flake.lock`.